lib/spark_api/models/virtual_tour.rb

```diff
@@ -13,6 +17,23 @@ def unbranded?
         attributes["Type"] == "unbranded"
       end
 
+      def url
+        attributes['Uri']
+      end
+
+      def description
+        attributes['Name']
+      end
+
+      def display_image
+        begin
+          response = Faraday::Connection.new.get(self.Uri) { |request| request.options.timeout = 20 }
+          open_graph = OGP::OpenGraph.new(response.body)
+          open_graph.image.url
+        rescue
+          return nil
+        end
+      end
```

Comment:
Let's talk tomorrow about this. We had simlar code elsewhere earlier and had to eliminate it. I have a few ideas if this is critical to the feature (even if not, I think it's cool and I'd like to keep it!)